lib/kino/terminator.ex

```diff
@@ -11,6 +11,45 @@ defmodule Kino.Terminator do
     {@name, node()}
   end
 
+  @doc """
+  Starts a Kino process to be shutdown by the terminator.
+  """
+  def start_child({mod, fun, args}, parent, gl) do
+    # We switch the group leader, so that the newly started
+    # process gets the same group leader as the caller
+    initial_gl = Process.group_leader()
+
+    Process.group_leader(self(), gl)
+
+    try do
+      {resp, pid} =
+        case apply(mod, fun, args) do
+          {:ok, pid} = resp -> {resp, pid}
+          {:ok, pid, _info} = resp -> {resp, pid}
+          resp -> {resp, nil}
+        end
+
+      if pid do
+        Kino.Bridge.reference_object(pid, parent)
+        Kino.Bridge.monitor_object(pid, cross_node_name(), {:terminate, pid}, ack?: true)
+      end
+
+      resp
+    after
+      Process.group_leader(self(), initial_gl)
+    end
+  end
+
+  @doc """
+  Starts a task that will terminate the parent in case of crashes.
+  """
+  def start_task(parent, fun) do
+    Task.start_link(fn ->
```

Comment:
We don't want to link, or am I missing it?

Reply:
This is being called inside the supervisor. We still want to link the task to its supervisor. :)

Reply:
Ahh sure it's for the child spec :D